contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
+
+The nightly release provides a mechanism to detect and resolve issues quickly and before they reach the more traffic-heavy _weekly_ release channels. It also serves to reduce the number of users affected by newly introduced issues
+
+The **nightly** release build is served to 0.05% of AMP traffic. It is possible to opt into the **nightly** channel via the `AMP_CANARY` cookie; see the [Opt-In Cookie section](#opt-in-cookie) for details.
+
+An additional **nightly-control** release channel exists, and is also served to 0.05% of AMP traffic. This release contains the same binaries as **stable**, and is used as a traffic-eqivalent baseline to compare **nightly** against for metrics and error reporting. It cannot be opted into.
+
+### Weekly
+
+The _weekly_ release channels are considered to be the primary "evergreen" release channels. The last **nightly** release from the previous week is promoted to the  **experimental** and **beta** release channels; the **beta** release from the previous week is promoted to the **stable** release channel.
+
+There are two sets of build configurations used in creating release builds: the _canary_ configuration and the _production_ configuration. The **experimental** and **beta** release channels are built off of the same commit, but with the _canary_ and _production_ configurations, respectively. The _canary_ configuration enables experimental components and features that may be turned off in _production_. Each is served to 0.5% of AMP traffic. It is possible to opt into the **experimental** or **beta** channels via the `AMP_CANARY` cookie; see the [Opt-In Cookie section](#opt-in-cookie) for details.
+
+The **stable** release is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (ignoring the possibility of [cherry-picks](#cherry-picks)).
+
+An additional **control** release channel exists, and is also served to 0.5% of AMP traffic. This release contains the same binaries as **stable**, and is used as a traffic-eqivalent baseline to compare **experimental** and **beta** against for metrics and error reporting. It cannot be opted into.
+
+See the [release cadence section](#release-cadence) for a more detailed breakdown of the release promotion cadence. To determine if a PR has been included in the current _weekly_ release, look for the GitHub labels _PR Use: In Canary_ or _PR Use: In Production_.
+
+### LTS
+
+The **lts** release channel provides a previous **stable** build for four-month intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all AMP partners, but is provided so partners performing a QA cycle may do so less often. The **lts** release must be explicitly opted into by adding `/lts` to the path of the runtime and extension scripts requested in the page:
```

Comment:
"Partners" is not necessarily clear for readers of this doc. AMP website developers maybe? Worth asking around for the best term to use here

Reply:
@CrystalOnScript @nainar What are your thoughts around the language here?

Reply:
I would just say "AMP users".
Maybe update to:
"The **lts** release channel allows QA cycles to be performed less often. It is not recommended for all AMP users."

Reply:
@CrystalOnScript I'm not a huge fan of "users" because that could also mean anyone with a web browser >__< - wdyt about "AMP publishers"? (no strong feelings if you think "users" still makes sense)

Reply:
@danielrozenberg - good point! I don't love `publishers` but I think it's better fitting than `users`. Maybe to make it specific:

"This is not recommended for all publishers building experiences with AMP, but is provided so that...."

Reply:
@CrystalOnScript I took a stab implementing your suggestion; would you mind taking a look at the latest version?